libdevice/cmake/modules/SYCLLibdevice.cmake

```diff
@@ -25,29 +29,17 @@ set(compile_opts
   -sycl-std=2017
   )
 
-if (WIN32)
-  set(devicelib-obj-file ${binary_dir}/libsycl-msvc.o)
-  add_custom_command(OUTPUT ${devicelib-obj-file}
-                     COMMAND ${clang} -fsycl -c
-                             ${compile_opts} ${sycl_targets_opt}
-                             ${CMAKE_CURRENT_SOURCE_DIR}/msvc_wrapper.cpp
-                             -o ${devicelib-obj-file}
-                     MAIN_DEPENDENCY msvc_wrapper.cpp
-                     DEPENDS wrapper.h device.h spirv_vars.h clang clang-offload-bundler
-                     VERBATIM)
-else()
-  set(devicelib-obj-file ${binary_dir}/libsycl-glibc.o)
-  add_custom_command(OUTPUT ${devicelib-obj-file}
-                     COMMAND ${clang} -fsycl -c
-                             ${compile_opts} ${sycl_targets_opt}
-                             ${CMAKE_CURRENT_SOURCE_DIR}/glibc_wrapper.cpp
-                             -o ${devicelib-obj-file}
-                     MAIN_DEPENDENCY glibc_wrapper.cpp
-                     DEPENDS wrapper.h device.h spirv_vars.h clang clang-offload-bundler
-                     VERBATIM)
-endif()
+set(devicelib-obj-file ${obj_binary_dir}/libsycl-crt.${lib-suffix})
```

Comment:
Now, if I build SYCL application with use of std functions on Windows I have to link against `libsycl-crt.obj`. On the other hand, when I build the same SYCL app on Linux I've got to use `libsycl-crt.o`. I think, this will confuse the user a bit.

Reply:
Hi, @s-kanaev 
Previously, If users want to use "assert" on Windows, they must use "libsycl-msvc.o" and If they want to use it on Linux, they must use "libsycl-glibc.o". Now, the device libraries' name are unified and I think ".obj" is more familiar to Windows users.
By the way, we are also working on the device library link by default feature in https://github.com/intel/llvm/pull/2400 , after the default link is finished, we will have nothing to worry about.
Thank you very much.

Reply:
Correct, we must not ask users to link any libraries explicitly.  This is a driver's job.  As soon as we provide a compiler option, it will not matter what the file names are.  The C/CXX standard library extension documentation will be updated, when we finalize the option.